### PR TITLE
refactor(esx_identity): jquery removal

### PR DESCRIPTION
**Pull request by MoskalykA**

### CODE REVIEW

File: [core]/esx_identity/html/css/style.css [hunk 1]
```diff
@@ -3,25 +3,27 @@
...
   width: 332px;
   opacity: 0.95;
-  position: absolute;
-  margin-left: auto;
-  margin-right: auto;
-  top: 30%;
   padding: 20px;
-  left: 50%; /* à 50%/50% du parent référent */
-  transform: translate(-50%); /* décalage de 50% de sa propre taille */
+
```
Comment (Flambaa) [inline [core]/esx_identity/html/css/style.css:17]: Empty Space

File: [core]/esx_identity/html/css/style.css [hunk 2]
```diff
@@ -3,25 +3,27 @@
...
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+
```
Comment (Flambaa) [inline [core]/esx_identity/html/css/style.css:22]: + Empty Space

**Review by Flambaa — CHANGES_REQUESTED** [review]

**Review by CsokiHUN — APPROVED** [review]
looks good, thanks

**Comment by Gellipapa** [comment]
@MoskalykA Why closed this pr?

**Comment by MoskalykA** [comment]
> @MoskalykA Why closed this pr?

I deleted the fork to make another pull request, I had forgotten that it delete the other pull requests

**Review by bitpredator — APPROVED** [review]

**Pull request merged**